pkg/template/template.go

```diff
@@ -7,6 +7,8 @@ import (
 	"text/template"
 
 	"github.com/Masterminds/sprig/v3"
+
+	"github.com/aquaproj/aqua/v2/pkg/asset"
```

Comment:
This doesn't actually work as is, because there's an import cycle, so setting as draft. But filing for food for thought anyway.